### PR TITLE
Using flag decorator in card table

### PR DESCRIPTION
**Pull request by DarkSide666**

### CODE REVIEW

File: src/Table/Column/Flag.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use Atk4\Data\Model;
+use Atk4\Ui\Table;
+
+/**
+ * Column for formatting country code as flags.
+ */
+class Flag extends Table\Column
+{
+    /** @var string Name of model field which contains country ALPHA-2 (2 letter) codes. */
+    public $code_field;
```
Comment (PhilippGrashoff) [inline src/Table/Column/Flag.php]: Didn't we switch to camelCase property names?

Comment (mvorisek) [reply src/Table/Column/Flag.php]: yes, we did everywhere!

unfortunatelly, PHP CS Fixer has no rule for it, so we need to rely on reviews, thanks!

File: src/Table/Column/Flag.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    public function getHtmlTags(Model $row, ?Field $field): array
+    {
+        $countryCode = $row->get($this->codeField);
+        $countryName = $this->nameField ? $row->get($this->nameField) : null;
+
+        return [
+            $field->shortName => $countryCode === null ? '' : $this->getApp()->getTag('i', [
+                'class' => strtolower($countryCode) . ' flag',
+                'title' => $countryCode . ($countryName === null ? '' : ' - ' . $countryName),
```
Comment (mvorisek) [inline src/Table/Column/Flag.php]: `$countryCode` should be in UC

File: src/Table.php [hunk 3]
```diff
@@ -482,7 +482,7 @@ public function renderRow(): void
                 if (!is_array($columns)) {
                     $columns = [$columns];
                 }
-                $field = !is_int($name) && $this->model->hasField($name) ? $this->model->getField($name) : null;
+                $field = is_int($name) ? null : $this->model->getField($name);
```
Comment (mvorisek) [inline src/Table.php]: ❤️

File: src/Table/Column/Flag.php (hunk 2 above)
Comment (DarkSide666) [reply src/Table/Column/Flag.php]: it is already in UC here in data, but to be sure I'll add strtoupper

**Comment by DarkSide666** [comment]
DO NOT MERGE THIS

**Comment by mvorisek** [comment]
resubmit as https://github.com/atk4/ui/pull/1959

**Pull request closed without merging**